pipeline-modules/deployment-service/gcp/private-helm.yaml

```
@@ -108,6 +108,7 @@ template: |
       {%- if helm_values_path != '' %}
       cp {{source_code_path}}/{{helm_values_path}} values_presub.yml
       cat values_presub.yml | envsubst > {{source_code_path}}/{{helm_values_path}}
+      rm -rf values_presub.yml
```

Since this is not directory, you shouldn't need the recursive flag, right?

```suggestion
      rm -f values_presub.yml
```